Cargo.toml

```diff
@@ -8,6 +8,7 @@ homepage = "https://github.com/ChrisMacNaughton/cargo-cacher"
 repository = "https://github.com/ChrisMacNaughton/cargo-cacher.git"
 
 [dependencies]
+cargo = "0.36.0"
```

Comment:
I wonder if [cargo-lock](https://github.com/RustSec/cargo-lock) might be a less invasive way to parse the Cargo.lock file. Pulling in cargo itself dramatically increases the build time of this crate.

Reply:
Done ! It is indeed much better.